logic/resolvable/ConcludableTest.java

```diff
@@ -310,4 +311,338 @@ public void test_conjunction_anonymous_variable_constraints_are_not_conflated()
             assertTrue(hasConcludable.has().attribute().isa().isPresent());
         });
     }
+
+    @Test
+    public void test_var_attribute_unification_satisfiable(){
+        Pair<String,String>[] concludableConclusionPairs = new Pair[]{
```

Comment:
We test the various cases for comparison based predicate (assignment, =, !=, <, >, <=, >= ) against a constant value "explicit-has"(?) conclusion.

Does the concludable pair.first() have a satisfiable 'unification' with the conclusion pair.second()?